app/views/handlers/list_repeating_block.py

```diff
@@ -1,27 +1,37 @@
+from typing import Any
+
 from flask import url_for
 
 from app.views.handlers.list_edit_question import ListEditQuestion
 
 
 class ListRepeatingBlock(ListEditQuestion):
+    def __init__(self, *args: Any) -> None:
+        super().__init__(*args)
```

Comment:
Add kwargs for safety?

Reply:
Instead of overriding the `__init__` just for an extra attr, I suggest just return what you need as a property or method.

```py
    @property
    def repeating_block_ids(self):
        return [block["id"] for block in self.parent_block["repeating_blocks"]]
```

Reply:
Added cached property good suggestion 👍